Documentation/gettingstarted.rst

```diff
@@ -935,7 +933,7 @@ After the script has successfully completed, you can log into the VM using
 
 All commands for the rest of the tutorial below should be run from inside this
 Vagrant VM.  If you end up disconnecting from this VM, you can always reconnect
-in a new terminal window just by running ``vagrant ssh`` again from the Cilium
+in a new terminal window just by running ``vagrant ssh`` again from the ``/cilium/examples/mesos``
```

Comment:
This path reference is a bit misleading as `/cilium/...` looks like an absolute path and people can't copy&paste it. I would write this as:

If you end up disconnecting from this VM, you can always reconnect by changing to the ``examples/mesos`` directory and then running the command ``vaggrant ssh``.

Reply:
definitely agree with thomas's comments re: absolute file paths.

Reply:
done.